<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ian-TECRA-M11" name="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest" tests="28" failures="0" timestamp="4 Jul 2014 23:33:07 GMT" time="41.861" errors="2">
  <testcase name="testgetBadgeDetailsWithMandatoryFields" time="2.985" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetCheckinDetailsWithOptionalFields" time="1.231" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testaddCheckinCommentNegativeParameters" time="1.069" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetFriendDetailsWithMandatoryFields" time="1.156" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testdeleteCheckinCommentNegativeParameters" time="2.261" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testapproveFriendsWithNegativeCase" time="1.177" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetFriendDetailsWithNegativeCase" time="1.157" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetTipsDetailsWithOptionlaFields" time="1.727" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetCheckinDetailsNegativeCase" time="1.146" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetCheckinDetailsWithMandatoryFields" time="1.806" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetMayorshipDetailsWithNegativeCase" time="1.140" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testunFriendsWithNegativeFields" time="1.159" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testdeleteCheckinCommentMandatoryParameters" time="2.288" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testaddCheckinCommentMandatoryParameters" time="1.158" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetCheckinRecentMandatoryParameters" time="1.807" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetUserDetailsWithMandatoryFields" time="1.194" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testaddCheckinsWithMandatoryParameters" time="3.618" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testaddCheckinsWithNegativeParameters" time="1.710" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetCheckinRecentNegativeParameters" time="1.108" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetMayorshipDetailsWithMandatoryFields" time="1.098" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetTipsDetailsWithNegativeCase" time="1.144" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest">
    <error type="java.lang.AssertionError" message="expected:&lt;400&gt; but was:&lt;200&gt;">
      <![CDATA[java.lang.AssertionError: expected:<400> but was:<200>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:365)
	at org.testng.Assert.assertEquals(Assert.java:375)
	at org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest.testgetTipsDetailsWithNegativeCase(FoursquareConnectorIntegrationTest.java:640)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1178)
	at org.testng.TestRunner.privateRun(TestRunner.java:757)
	at org.testng.TestRunner.run(TestRunner.java:608)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1158)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1083)
	at org.testng.TestNG.run(TestNG.java:999)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:178)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
    </error>
  </testcase>
  <testcase name="testapproveFriendsWithMandatoryFields" time="0.591" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest">
    <error type="java.lang.AssertionError" message="expected:&lt;200&gt; but was:&lt;400&gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<400>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:365)
	at org.testng.Assert.assertEquals(Assert.java:375)
	at org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest.testapproveFriendsWithMandatoryFields(FoursquareConnectorIntegrationTest.java:667)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1178)
	at org.testng.TestRunner.privateRun(TestRunner.java:757)
	at org.testng.TestRunner.run(TestRunner.java:608)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1158)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1083)
	at org.testng.TestNG.run(TestNG.java:999)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:178)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
    </error>
  </testcase>
  <testcase name="testgetUserDetailsWithNegativeCase" time="1.105" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetBadgeDetailsWithNegativeCase" time="1.100" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testunFriendsWithMandatoryFields" time="0.579" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetTipsDetailsWithMandatoryFields" time="1.253" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetFriendDetailsWithOptionalFields" time="1.109" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
  <testcase name="testgetCheckinRecentOptionalParameters" time="2.985" classname="org.wso2.carbon.connector.integration.test.foursquare.FoursquareConnectorIntegrationTest"/>
</testsuite>
